.github/actions/remove-dependabot-semver-labels/action.yml

```
@@ -0,0 +1,74 @@
+name: 'Remove Dependabot Semver Labels'
+description: |
+  Removes auto-added semver labels (major/minor/patch) from Dependabot PRs.
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)